DEVELOPMENT.md

```diff
@@ -10,21 +10,18 @@ to `Knative Serving`. Also take a look at:
 
 ## Getting started
 
-1. Create [a GitHub account](https://github.com/join)
-1. Setup [GitHub access via
-   SSH](https://help.github.com/articles/connecting-to-github-with-ssh/)
-1. Install [requirements](#requirements)
-1. [Set up a kubernetes cluster](./docs/creating-a-kubernetes-cluster.md)
-1. [Set up a docker repository you can push
-   to](./docs/setting-up-a-docker-registry.md)
-1. Set up your [shell environment](#environment-setup)
-1. [Create and checkout a repo fork](#checkout-your-fork)
-
-Once you meet these requirements, you can [start Knative Serving](#starting-knative-serving)!
+Follow the instructions below to set up your development environment.  Once you
+meet these requirements, you can
+[start Knative Serving](#starting-knative-serving)!
 
 Before submitting a PR, see also [CONTRIBUTING.md](./CONTRIBUTING.md).
 
-### Requirements
+### Prerequisites
+
+Start by creating [a GitHub account](https://github.com/join), then setup
+[GitHub access via SSH](https://help.github.com/articles/connecting-to-github-with-ssh/).
+
+### Install requirements <a name="requirements"></a>
```

Comment:
You don't need anchors here; GitHub will generate them automatically (e.g. as `#install-requirements`).

Reply:
I'm adding the anchors to preserve the previous names in case there are existing links to them (this section used to just be called "Requirements"). Is that not necessary?

Reply:
Aha, I see. This seems fine to avoid needing to find all the references, though I'd also be fine with breaking anchor links in the development section.